opentracing/src/main/java/opentracing/propagation/SplitTextFormatInjector.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package opentracing.propagation;
+
+import java.util.Map;
+
+/**
+ *
+ */
+interface SplitTextFormatInjector extends Injector<Map> {
```

Comment:
@michaelsembwever it's not a niche example... you have previously raised your concern (which I agree with, btw) about the potentially unbounded cost of the Baggage feature in a distributed system... owners of said distributed systems should be able to strip out baggage without requiring changes to the OT implementation they're using, and that's something the `Split*` carriers accomplish.

(There is also the more esoteric but nevertheless real requirement from Uber regarding the separation of the fixed-width tracer state from the baggage, but that is secondary in my mind)